service/src/main/java/org/kiwiproject/champagne/resource/AuditRecordResource.java

```diff
@@ -0,0 +1,43 @@
+package org.kiwiproject.champagne.resource;
+
+import static org.kiwiproject.search.KiwiSearching.zeroBasedOffset;
+
+import javax.annotation.security.PermitAll;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.DefaultValue;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.QueryParam;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+import org.kiwiproject.champagne.dao.AuditRecordDao;
+import org.kiwiproject.spring.data.KiwiPage;
+
+import com.codahale.metrics.annotation.ExceptionMetered;
+import com.codahale.metrics.annotation.Timed;
+
+import lombok.AllArgsConstructor;
+
+@Path("/audit")
+@Consumes(MediaType.APPLICATION_JSON)
+@Produces(MediaType.APPLICATION_JSON)
+@PermitAll
+@AllArgsConstructor
+public class AuditRecordResource {
+
+    private final AuditRecordDao auditRecordDao;
+
+    @GET
+    @Timed
+    @ExceptionMetered
+    public Response getPagedAudits(@QueryParam("pageNumber") @DefaultValue("1") int pageNumber,
```

Comment:
Is it worth returning a 400 if we get a `pageNumber=0` ? Or no?

Reply:
For now, I think we are ok